activerecord/test/schema/mysql2_specific_schema.rb

```diff
@@ -30,9 +30,9 @@
   end
 
   create_table :key_tests, force: true, options: "ENGINE=MyISAM" do |t|
-    t.string :awesome
-    t.string :pizza
-    t.string :snacks
+    t.string :awesome, limit: 250
+    t.string :pizza, limit: 250
+    t.string :snacks, limit: 250
```

Comment:
Do we need the limit, since max key length is now 3072 bytes?

Reply:
There is another limit for MyISAM storage engine since it uses `ENGINE=MyISAM` option. 
https://dev.mysql.com/doc/refman/5.7/en/myisam-storage-engine.html
> The maximum key length is 1000 bytes. This can also be changed by changing the source and recompiling. For the case of a key longer than 250 bytes, a larger key block size than the default of 1024 bytes is used.

Another idea is removing `ENGINE=MyISAM` option.
This table is created for `type: :fulltext` since MySQL used to support FULLTEXT option only with MyISAM storage engine. Starting from MySQL 5.6, InnoDB storage engine also supports  FULLTEXT option. Then we may not need `ENGINE=MyISAM` option and `limit: 250` anymore.

Reply:
Removed `ENGINE=MyISAM` option and `limit: 250`.

Reply:
Ah yes, I didn't notice MyISAM! Glad we can remove it.